### DISCUSSION TITLE
How to load default values into form [Question]

### DISCUSSION THREAD

**Discussion by chiester** [vazco/uniforms#1016] [Q&A] [answered]
My apologies for beginner questions in the issues. 

How do I use Uniforms to preload the form with values? Like if I want to use it as an update form, I fetch data from the back end and then the user can update their saved values, but I still want all the advantages of validation, etc. ?

**Comment by wadamek65** [comment, marked as answer]
Hi @chiester ,

You can do it in a few ways, two most commonly used are:
1. Pass a `model` prop with your values to the form that you are using. As long as the properties passed exist in the schema, the inputs will get this value prefilled.
2. Depending on the bridge you are using, you can also define default values in the schema. For example, simple schema has a `defaultValue` keyword that you can use.

Feel free to create a discussion/issue with any questions and we will do our best to help.
